src/components/create-journey/CreateJourney.js

```diff
@@ -20,39 +44,25 @@ class CreateJourney extends Component {
     };
 
     /**
-     * Handle change on the input field
-     * @param {} e 
+     * Save the value of the field when it changes
      */
-    handleChange = function(e) {
-        // console.log(e.target)
-        switch(e.target.name){
-            case 'journey-id':
-                this.setState({ journeyId: e.target.value });
-                break;
-            case 'destination':
-                this.setState({ destination: e.target.value });
-                break;
-            case 'price':
-                this.setState({ price: e.target.value });
-                break;
-            default:
-                console.error('no such state attribut')
-                break;
-        }
-    }.bind(this);
+    handleChange = name => event => {
```

Comment:
:flushed:

Reply:
I can't answer you, I don't remember why I did it like this.

Reply:
please put a name 
remove the firsrt argument passed 
replace the name in the handleChange function for name.target.name